packages/lab/src/radio-button/RadioButtonIcon.css

```diff
@@ -1,84 +1,76 @@
 }
 
-/* Styles applied to RadioButtonIcon border if `disabled={true}` or in a disabled FormField */
-.saltRadioButton-disabled .saltRadioButtonIcon-border,
-.saltFormField-disabled .saltRadioButton .saltRadioButtonIcon-border,
-.saltRadioButton-disabled:hover .saltRadioButtonIcon-border,
-.saltFormField-disabled .saltRadioButton:hover .saltRadioButtonIcon-border {
+/* Styles applied to RadioButtonIcon border if disabled */
+.saltRadioButtonIcon-disabled .saltRadioButtonIcon-border,
+.saltRadioButton:hover .saltRadioButtonIcon-disabled .saltRadioButtonIcon-border {
   fill: var(--saltRadioButton-icon-fill-disabled, var(--radioButton-icon-fill));
-  stroke: var(--saltRadioButton-icon-strokeColor-disabled, var(--radioButton-icon-strokeColor-disabled));
+  stroke: var(--saltRadioButton-icon-strokeColor-disabled, var(--salt-selectable-borderColor-selectedDisabled));
 }
 
-/* Styles applied to RadioButtonIcon border if `checked={true}` when `disabled={true}` or in a disabled FormField  */
-.saltFormField-disabled .saltRadioButton .saltRadioButtonIcon-checked .saltRadioButtonIcon-border,
-.saltFormField-disabled .saltRadioButton:hover .saltRadioButtonIcon-checked .saltRadioButtonIcon-border,
-.saltRadioButton-disabled .saltRadioButtonIcon-checked .saltRadioButtonIcon-border,
-.saltRadioButton-disabled:hover .saltRadioButtonIcon-checked .saltRadioButtonIcon-border {
+/* Styles applied to RadioButtonIcon border if checked and disabled */
+.saltRadioButtonIcon-disabled.saltRadioButtonIcon-checked .saltRadioButtonIcon-border,
+.saltRadioButton:hover .saltRadioButtonIcon-disabled.saltRadioButtonIcon-checked .saltRadioButtonIcon-border {
   fill: var(--saltRadioButton-icon-fill-checked-disabled, var(--radioButton-icon-fill-checked-disabled));
   stroke: var(--saltRadioButton-icon-strokeColor-checked-disabled, transparent);
 }
 
-/* Styles applied to inner circle when `checked={true}` */
+/* Styles applied to inner circle when checked */
 .saltRadioButtonIcon-inner-checked {
   fill: var(--saltRadioButton-icon-inner-fill, var(--salt-selectable-primary-foreground-selected));
 }
+
+/* Styles applied to radio button icon with error */
+.saltRadioButton:hover .saltRadioButtonIcon-error .saltRadioButtonIcon-border,
+.saltRadioButtonIcon-error,
+.saltRadioButtonIcon-error .saltRadioButtonIcon-border {
+  stroke: var(--saltRadioButton-icon-strokeColor-error, var(--salt-selectable-borderColor-error));
+}
+
+/* Styles applied to radio button icon when checked and with error */
+.saltRadioButton:hover .saltRadioButtonIcon-checked.saltRadioButtonIcon-error .saltRadioButtonIcon-border,
+.saltRadioButtonIcon-checked.saltRadioButtonIcon-error .saltRadioButtonIcon-border {
+  fill: var(--saltRadioButton-icon-fill-error, var(--salt-selectable-borderColor-error));
+}
+
+/* Styles applied to radio button icon when checked and with error */
+.saltRadioButton-disabled .saltRadioButtonIcon-error {
```

Comment:
can it be error and disabled? i wouldn't think so and can't see in specs
if it is, we shouldn't use foundation directly, need a status-error-border-disabled token in that case

Reply:
i saw your comment in figma asking, maybe remove for now?